pkgs/top-level/all-packages.nix

```diff
@@ -17207,6 +17207,8 @@ in
 
   camingo-code = callPackage ../data/fonts/camingo-code { };
 
+  chili-theme = libsForQt5.callPackage ../data/themes/chili/chili.nix {};
```

Comment:
How about use default.nix instead of chili.nix? Other files under theme dir use default.nix